OpenOversight/app/main/views.py

```diff
@@ -632,6 +632,39 @@ def delete_tag(tag_id):
     return redirect(url_for('main.index'))
 
 
+@main.route('/tag/set_featured/<int:tag_id>', methods=['POST'])
+@login_required
+@ac_or_admin_required
+def set_featured_tag(tag_id):
+    tag = Face.query.filter_by(id=tag_id).first()
+
+    if not tag:
+        flash('Tag not found')
```

Comment:
Is there a standard for doing a 404 vs doing a flash and redirect?

Reply:
You're right I think it should be a 404. I used the delete_tag() function as a template, but that should probably be a 404 as well. I'll change that.